src/components/tooltip/TooltipDirective.js

```diff
@@ -490,6 +490,12 @@ goog.require('ga_topic_service');
               var name = feature.get('name');
               var featureId = feature.getId();
               var layerId = feature.get('layerId') || layer.get('bodId');
+              if (layer.get('type') == 'KML') {
+                layerId = layer.label;
+                if (name && name.length) {
```

Comment:
@cedricmoullet For KML, and if the feature has a 'name' attribute, it will be the name.